libpod/define/container_inspect.go

```diff
@@ -229,6 +229,7 @@ type InspectContainerState struct {
 	RestoreLog     string             `json:"RestoreLog,omitempty"`
 	Restored       bool               `json:"Restored,omitempty"`
 	StoppedByUser  bool               `json:"StoppedByUser,omitempty"`
+	IntelRdtClosID string             `json:"IntelRdtClosID,omitempty"`
```

Comment:
This should go in HostConfig, not ContainerState

Reply:
done